### PR TITLE
[RA1] Proofreading all chapters

### PR DESCRIPTION
**Pull request by karinesevilla**
Tables adjustments for pdf better rendering
Typos
References : external and internal links, tables

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 1]
```diff
@@ -754,21 +754,24 @@ Confidentiality and Integrity Requirements
...
+       Integrity
      - The Platform **must** support Confidentiality and Integrity of process-
        related metadata and restrict information sharing with only the
        process owner (e.g., tenant)
      - :ref:`ref_arch/openstack/chapters/chapter06:confidentiality and
        integrity`
    * - sec.ci.006
-     - Confidentiality/Integrity
+     - Confidentiality/
+       Integrity
```
[the remark this answers is not in the document]
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.rst:770]: @collivier I have  added a line, could you check if it is the good rendering.

File: doc/ref_arch/openstack/chapters/chapter04.rst [hunk 2]
```diff
@@ -1650,34 +1664,34 @@ choices.
...
+Reference Architecture
+<https://fuel-ccp.readthedocs.io/en/latest/design/ref_arch_100_nodes.html#services-placement-summary>`__
 for 100, 300 and 500 nodes will help in deciding which OpenStack and
 other services (such as database, messaging) run on which nodes in what
 Cloud Centre and the number of copies that should be deployed. These
 references also present the pros and cons of DCP and CCP and designs to
 address some of the challenges of each of the models.
 
-Table 8-4 in the Reference Model
+In the Reference Mode, the
```
Comment (ildikov) [inline doc/ref_arch/openstack/chapters/chapter04.rst:1684]:
```suggestion
In the Reference Model, the
```

**Review by ildikov — COMMENTED** [review]
Hi Karine,

I found one typo in line. Looks good otherwise.

Thanks,
Ildikó

**Comment by karinesevilla** [comment]
Following PR #3035 merge, conflicts fixed

**Review by collivier — APPROVED** [review]

**Pull request merged**